### PR TITLE
Implement Alt Assets Variable LUS

### PR DESCRIPTION
**Pull request by Malkierian**
Bump LUS to `94b755b77c0c0f566686c9d23fa4c15e23473a0c`.
Remove `CVAR_ALT_ASSETS` from LUS cvars cmake.
Change all pertinent `CVarGetInteger` calls for `CVAR_ALT_ASSETS` to `ResourceManager::IsAltAssetsEnabled` or "gAltAssets" as appropriate.
Implement frame-end toggle of `mAltAssetsEnabled`.

### Build Artifacts
  - [2ship-linux.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/1556558481.zip)
  - [2ship-mac.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/1556562956.zip)
  - [2ship-windows.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/1556572075.zip)

### CODE REVIEW

File: mm/2s2h/BenPort.cpp [hunk 1]
```diff
@@ -649,13 +651,23 @@ extern "C" void Graph_StartFrame() {
...
         }
     }
 #endif
 
+    bool curAltAssets = CVarGetInteger("gAltAssets", 0);
+    if (prevAltAssets != curAltAssets) {
+        prevAltAssets = curAltAssets;
+        Ship::Context::GetInstance()->GetResourceManager()->SetAltAssetsEnabled(curAltAssets);
+        gfx_texture_cache_clear();
+        // TODO: skeleton patch, hooks
```
Comment (garrettjoecox) [inline mm/2s2h/BenPort.cpp]: Does this replace the commented code down in `Graph_ProcessGfxCommands`?

Comment (Malkierian) [reply mm/2s2h/BenPort.cpp]: Oh, that is where that's supposed to go. Didn't have the `ShouldClearTextureCacheAtEndOfFrame` stuff in SoH, though.

Comment (Malkierian) [reply mm/2s2h/BenPort.cpp]: Oh, that's just the variable name. Hold on.

Comment (Malkierian) [reply mm/2s2h/BenPort.cpp]: Should I leave `gfx_texture_cache_clear` commented for now, then?

Comment (garrettjoecox) [reply mm/2s2h/BenPort.cpp]: That'd be an @Archez question

File: mm/2s2h/BenPort.cpp [hunk 2]
```diff
@@ -148,6 +149,7 @@ OTRGlobals::OTRGlobals() {
     // tell LUS to reserve 3 SoH specific threads (Game, Audio, Save)
     context =
         Ship::Context::CreateInstance("2 Ship 2 Harkinian", appShortName, "2ship2harkinian.json", archiveFiles, {}, 3);
+    prevAltAssets = CVarGetInteger("gAltAssets", 0);
```
Comment (Archez) [inline mm/2s2h/BenPort.cpp:152]: Looks like we are missing the complimentary initial set asset mode on the resource manager that you added on the SoH side.

```suggestion
    prevAltAssets = CVarGetInteger("gAltAssets", 0);
    context->GetResourceManager()->SetAltAssetsEnabled(prevAltAssets);
```

**Review by Archez — APPROVED** [review]

**Pull request merged**